app/api/v1/auth.py

```
@@ -82,18 +87,58 @@ async def login(
             LoginInput(email=payload.email, password=payload.password)
         )
         
-        token = token_svc.create_access_token(subject=user.id)
-        return Token(access_token=token, token_type="bearer")
+        access_token = token_svc.create_access_token(subject=user.id)
+        refresh_token = token_svc.create_refresh_token(user_id=user.id)
+        
+        return Token(
+            access_token=access_token, 
+            refresh_token=refresh_token,
+            token_type="bearer",
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to argument: "token_type"

(S106)